tools/scripts/utils.js

```diff
@@ -132,7 +132,7 @@ const createBmpRange = (r, { addBrackets } = { addBrackets: true }) => {
 
 const assemble = ({ name, alias, codePoints }) => {
     const { bmp, astral, isBmpLast } = createRange(codePoints);
-    const result = { name };
+    const result = { name: name.replace('_And_', '_and_').replace('_For_', '_for_') };
```

Comment:
FWIW, I’m against this change. IMHO we should use [the canonical names](https://unicode.org/Public/10.0.0/ucd/PropertyValueAliases.txt), not an arbitrarily-cased variation of them. I understand the desire to merge this with the workaround but I’m hoping it can be removed later.

Reply:
Absolutely, we should ultimately use the canonical names :) I just want to keep the refactoring separate from behavioral changes.